receiver/k8seventsreceiver/receiver_test.go

```diff
@@ -149,6 +151,8 @@ func getEvent() *corev1.Event {
 			Name:              "1",
 			Namespace:         "test",
 			CreationTimestamp: v1.Now(),
+			Labels:            make(map[string]string),
```

Comment:
Shall we add some tests that will also illustrate what type of data this feature produces?

Reply:
Sure, added TestEventLabelsAndAnnotationsExtractionOptions test